packages/lib/src/vis/line/errorBarsGeometry.ts

```diff
@@ -31,10 +31,9 @@ class ErrorBarsGeometry extends H5WebGeometry<'position', Params> {
     } = this.params!;
 
     const value = ordinates[index];
-    const isIgnored = ignoreValue ? ignoreValue(value) : false;
     const bufferIndex = index * 2;
 
-    if (isIgnored) {
```

Comment:
I liked the intermediate variable: brain computing `.?` can be tricky sometime :sweat_smile:

Reply:
Haha, I personally found the ternary more difficult to read. :sweat_smile: With `?.`, the fact that `ignoreValue` is optional doesn't distract as much from the main condition, which is whether or not the value is to be ignored.